DigitalLearningSolutions.Data.Tests/Services/UserServiceTests.cs

```diff
@@ -458,5 +458,53 @@ public void NewEmailAddressIsValid_returns_true_for_delegate_only_with_unchanged
             A.CallTo(() => userDataService.GetAdminUserByEmailAddress(email)).MustNotHaveHappened();
             A.CallTo(() => userDataService.GetDelegateUsersByEmailAddress(email)).MustNotHaveHappened();
         }
+
+        [Test]
+        public void IsEmailValidForCentre_should_return_false_if_user_at_centre_has_email()
+        {
+            // Given
+            const string email = "email@test.com";
+            A.CallTo(() => userDataService.GetAdminUserByEmailAddress(email)).Returns(null);
+            A.CallTo(() => userDataService.GetDelegateUsersByEmailAddress(email)).Returns
+                (new List<DelegateUser> { UserTestHelper.GetDefaultDelegateUser(3, emailAddress: email, centreId: 3) });
+
+            // When
+            var result = userService.IsEmailValidForCentre(email, 3);
+
+            // Then
+            result.Should().BeFalse();
+        }
+
+        [Test]
+        public void IsEmailValidForCentre_should_return_true_if_user__not_at_centre_has_email()
+        {
+            // Given
+            const string email = "email@test.com";
+            A.CallTo(() => userDataService.GetAdminUserByEmailAddress(email)).Returns(null);
+            A.CallTo(() => userDataService.GetDelegateUsersByEmailAddress(email)).Returns

```

Comment:
Just realised with the test - why are we getting admin email addresses? Should we only call get delegate users and make sure the method name reflects that we're checking delegates specifically?

Reply:
Just because that was what the checking in the delegate registration journey were doing. I agree it is a bit pointless now I'm refactoring.